app/src/main/java/org/jellyfin/mobile/utils/SystemUtils.kt

```diff
@@ -115,6 +117,14 @@ private fun Context.downloadFile(request: DownloadManager.Request, @DownloadMeth
 
 fun Activity.isAutoRotateOn() = Settings.System.getInt(contentResolver, ACCELEROMETER_ROTATION, 0) == 1
 
+fun Fragment.isPackageInstalled(packageName: String): Boolean {
+    return try {
+        packageName.isNotEmpty() && requireContext().packageManager.getApplicationInfo(packageName, 0).enabled
+    } catch (e: PackageManager.NameNotFoundException) {
+        false
+    }
+}
```

Comment:
```suggestion
fun Fragment.isPackageInstalled(packageName: String) = try {
    packageName.isNotEmpty() && requireContext().packageManager.getApplicationInfo(packageName, 0).enabled
} catch (e: PackageManager.NameNotFoundException) {
    false
}
```

This is just a personal preference but you can write it like this